core/src/mindustry/graphics/MenuRenderer.java

```diff
@@ -42,7 +42,7 @@ public MenuRenderer(){
     private void generate(){
         world.beginMapLoad();
         Tiles tiles = world.resize(width, height);
-        Seq<Block> ores = content.blocks().select(b -> b instanceof OreBlock);
+        Seq<Block> ores = content.blocks().select(OreBlock.class::isInstance);
```

Comment:
this is awful, never use reflection in a place where standard java works just fine